tests/sampling/test_jax.py

```diff
@@ -459,3 +464,22 @@ def test_idata_contains_stats(sampler_name: str):
     for stat_var, stat_var_dims in stat_vars.items():
         assert stat_var in stats.variables
         assert stats.get(stat_var).values.shape == stat_var_dims
+
+
+def test_sample_numpyro_nuts_block_adapt():
+    with pm.Model(
+        coords=dict(level=["Basement", "Floor"], county=[1, 2]),
+    ) as model:
+        # multilevel modelling
+        a = pm.Normal("a")
+        s = pm.HalfNormal("s")
+        a_g = pm.Normal("a_g", a, s, dims="level")
+        s_g = pm.HalfNormal("s_g")
+        a_ig = pm.Normal("a_ig", a_g, s_g, dims=("county", "level"))
+        trace = sample_numpyro_nuts(
+            nuts_kwargs=dict(
+                dense_mass=[
+                    ("a", "a_g"),
+                ]
+            )
+        )
```

Comment:
```suggestion
        trace = sample_numpyro_nuts(
            tune=10,
            draws=10,
            nuts_kwargs=dict(
                dense_mass=[
                    ("a", "a_g"),
                ]
            )
        )
```

Reply:
Is there anything useful we can assert?

Reply:
I can imagine `mock.spy` on numpyro internals but since it is a user facing kwarg, it should just pass